pallets/rmrk-core/src/tests.rs

```diff
@@ -843,21 +843,21 @@ fn lookup_root_owner_nesting_budget_works() {
 		);
 
 		let budget = budget::Value::new(<Test as Config>::NestingBudget::get());
-		// This should also fail since we nested too many nfts.
+		// This should also fail since we are not allowed to nest too many NFTs
 		assert_noop!(
 			RMRKCore::lookup_root_owner(
 				COLLECTION_ID_0,
 				<Test as Config>::NestingBudget::get() + 1,
 				&budget
 			),
-			Error::<Test>::TooManyRecursions
+			Error::<Test>::NoAvailableNftId
 		);
```

Comment:
This test is not failing because the NFT is too deeply nested, it is failing because it doesn't even exist, hence we get the `NoAvailableNftId`. But since it is not possible anymore to too deeply nest NFTs I think we don't need this check.

Reply:
The check is in case a future change happens that would re-allow deeply nesting too many NFTs. This is more of a preventative test case.